service/perf/metrics.go

```diff
@@ -71,10 +72,21 @@ func NewMetrics(namespace string, registry *prometheus.Registry) *Metrics {
 			Name:      "sessions_total",
 			Help:      "Total number of active RTC sessions",
 		},
-		[]string{"groupID", "callID"},
+		[]string{"groupID"},
```

Comment:
Not sure why I added `callID` in the first place as it's not a very useful label. Now only considering groups (aka clients).